pages/maintainers.md

```diff
@@ -31,9 +36,12 @@ This document lists community members who are [repository maintainers](/pages/go
   - [Dualization.jl](https://github.com/jump-dev/Dualization.jl)
 * Jack Dunn (@JackDunnNZ)
   - [AmplNLWriter.jl](https://github.com/jump-dev/AmplNLWriter.jl)
+* James Foster (@jd-foster)
+  - [JuMP.jl](https://github.com/jump-dev/JuMP.jl)
```

Comment:
I'll wait for @jd-foster to accept the invite before merging this.

Reply:
Thank you Oscar for the invitation to be a maintainer! I have accepted.